velox/docs/functions/iceberg/functions.rst

```diff
@@ -22,3 +22,23 @@ Refer to `Iceberg documenation <https://iceberg.apache.org/spec/#partition-trans
    Supported types for ``input`` are INTEGER, BIGINT, DECIMAL, DATE, TIMESTAMP, VARCHAR, VARBINARY. ::
        SELECT bucket(128, 'abcd'); -- 4
        SELECT bucket(100, 34L); -- 79
+
+.. iceberg:function:: truncate(width, input) -> same type as input
+   Returns the truncated value of the input based on the specified width.
+   For numeric values, truncate to the nearest lower multiple of ``width``, the truncate function is: input - (((input % width) + width) % width).
+   The ``width`` is used to truncate decimal values is applied using unscaled value to avoid additional (and potentially conflicting) parameters.
+   For string values, it truncates a valid UTF-8 string with no more than ``width`` code points.
+   In contrast to strings, binary values do not have an assumed encoding and are truncated to ``width`` bytes.
+
+   Argument ``width`` must be a positive integer.
+   Supported types for ``input`` are: SHORTINT, TYNYINT, SMALLINT, INTEGER, BIGINT, DECIMAL, VARCHAR, VARBINARY. ::
+
+       SELECT truncate(10, 11); -- 10
+       SELECT truncate(10, -11); -- -20
+       SELECT truncate(7, 22); -- 21
+       SELECT truncate(0, 11); -- throw
```

Comment:
throw -> error: <error message>

Let's include error message to illustrate what kind of error is expected